### PR TITLE
Feat/initial setup apicalls

### LINKED ISSUE #2: Fetch from 4 API endpoints

### PR DESCRIPTION
**Pull request by lkarpins**
** Resolves #2 ,  #4 , #5, #6, #7, #8 **

** Describe the functionality to be tested: **
- NavBar // Footer Components Created/Styled
- Routing established between pages
- Wired apiCalls with useEffect
- Added images to assets folder

** Giphy of how you feel submitting this PR: **

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,12 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "normalize.css": "^8.0.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^5.3.0",
     "react-scripts": "5.0.1",
+    "sass": "^1.54.5",
```
Comment (irmakerdem) [inline package.json:14]: sassy 💅

**Comment by Rallen13** [comment]
Great job driving Lidia!

File: src/Components/Routes/Routes.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Route, Switch } from "react-router-dom";
+import HomeView from "../../Views/HomeView/HomeView";
+import AboutView from "../../Views/AboutView/AboutView";
+import ClinicsView from "../../Views/ClinicsView/ClinicsView";
+import StatePoliciesView from "../../Views/StatePoliciesView/StatePoliciesView";
+
+const Routes = () => {
```
Comment (catlynbowles) [inline src/Components/Routes/Routes.js:8]: I love the use of this Routes component !

File: src/Components/Footer/Footer.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
+.footer-logo-container {
+  background-color: #3c3525;
+  padding: 1.5rem;
```
Comment (irmakerdem) [inline src/Components/Footer/Footer.scss:3]: Nice use of rem to make the styling responsive! 🥳

File: src/utils/apiCalls.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+export const getAllStatePolicies = async () => {
```
Comment (catlynbowles) [inline src/utils/apiCalls.js:1]: Yay async! 🌺

**Review by catlynbowles — APPROVED** [review]

File: src/Views/ClinicsView/ClinicsView.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  getAllMisinformationClinics,
+  getAllAuthorizedClinics,
+} from "../../utils/apiCalls";
+
+const ClinicsView = () => {
+  const [authorizedClinics, setAuthorizedClinics] = useState([]);
+  const [misinformationCenters, setMisinformationCenters] = useState([]);
+  const [resources, setResources] = useState([]);
+
+  useEffect(() => {
```
Comment (irmakerdem) [inline src/Views/ClinicsView/ClinicsView.js:14]: HOOK! 🪝

File: src/Views/StatePoliciesView/StatePoliciesView.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { useEffect, useState } from "react";
+import { getAllStatePolicies } from "../../utils/apiCalls";
+
+const StatePoliciesView = () => {
+  const [policies, setPolicies] = useState([]);
+
+  useEffect(() => {
+    getAllStatePolicies().then((data) => setPolicies(data.data.state));
```
Comment (irmakerdem) [inline src/Views/StatePoliciesView/StatePoliciesView.js:9]: Just realizing that there is `.state` which isn't the JS state but the US state ahahah 🤡

File: src/utils/apiCalls.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+  const URL = "https://roed-to-aide-be.herokuapp.com/api/v1/states";
+  const response = await fetch(URL, { method: "GET" });
```
Comment (irmakerdem) [inline src/utils/apiCalls.js:3]: async await 😎

**Review by irmakerdem — APPROVED** [review]
SO MUCH PROGRESS!!! BUSY BEES 🐝

**Pull request merged**